### DISCUSSION TITLE
What MQTT topic do I need to listen for Z-Wave button presses?

### DISCUSSION THREAD

**Discussion by sofakng** [zwave-js/zwave-js-ui#1789] [Q&A]
I'm using ZWAVEJS2MQTT and have a Z-Wave light switch.

What MQTT topic is published when I press the up/down buttons of the light switch?

I'd like to configure Tasmota to listen to this MQTT topic and turn on it's connected RGBW bulb.

I've been told to enable [Scene Control] on my Z-Wave switch.  Is this the correct?  Do I then need to monitor the MQTT topic /zwave/_EVENTS/ZWAVE_GATEWAY-zwavejs2mqtt/node/node_value_updated?

I was thinking maybe there is something in /zwave/<Room>/<Device>/keyPressed(?) would exist?

**Comment by robertsLando** [comment]
Do you see a value id in the UI that shows that? You could also use mqtt explorer to see values sent via mqtt, if it is a notification there is a special topic for them, you find everything you need under mqtt section in docs: https://zwave-js.github.io/zwavejs2mqtt/#/guide/mqtt?id=using-mqtt

**Reply by sofakng** [reply]
Sorry, but I'm not sure what you mean.  I'm not familiar enough with Z-Wave to understand what is an event or a notification.

Where could I find this information?

I was thinking it wouldn't be too difficult to detect a button press from the switch?

**Reply by robertsLando** [reply]
@sofakng I don't know what kind of notification comes when you press the button, you should try to check logs when you press it and you will see the event if it is catched (and also the associated mqtt publish

**Reply by Westcott1** [reply]
I'm also looking into this in discussion topic 1941

**Reply by robertsLando** [reply]
Link is broken

**Reply by Westcott1** [reply]
Sorry, I don't know how to post links to topics.
